### PR TITLE
Annotation to version attribute

### PR DESCRIPTION
**Pull request by trurlurl**
Responds to issue #767.

### CODE REVIEW

**Comment by skinkie** [comment]
I think the wording is still ambiguous.

File: xsd/netex_framework/netex_responsibility/netex_version_support.xsd [hunk 1]
```diff
@@ -186,7 +186,7 @@ Rail transport, Roads and Road transport
 		</xsd:attribute>
 		<xsd:attribute name="version" type="VersionIdType" use="optional">
 			<xsd:annotation>
-				<xsd:documentation>Version number of entity.</xsd:documentation>
+				<xsd:documentation>Version number of entity. Allowed values are "any", or a string adhering to agreed-upon formatting and interpretation rules (representing, e.g., numbers or dates). Consecutive versions should be strictly increasing.</xsd:documentation>
```
Comment (trurlurl) [inline xsd/netex_framework/netex_responsibility/netex_version_support.xsd:189]:
```suggestion
				<xsd:documentation>Version number of entity. The value may be "any" or a VERSION number adhering to agreed-upon formatting and interpretation rules (representing, e.g., a number or a date). Consecutive VERSION numbers should be strictly increasing.</xsd:documentation>
```

**Review by skinkie — APPROVED** [review]

**Review by ue71603 — APPROVED** [review]

**Comment by skinkie** [comment]
@ue71603 maybe to master too?

**Comment by ue71603** [comment]
@trurlurl can you do it to master too?

**Pull request merged**